### ISSUE TITLE
[V2] Suggestion: Adding skipZeroWeight and skipInactive parameters to GetActiveVolumes

### ISSUE THREAD

**Issue by AlexandreFiset** [Unity-Technologies/PostProcessing#399]
Hi!

Sometimes, it really is convenient to have a list of all currently active OR to-be active volumes. For instance, we use the PostProcessManager.instance.GetActiveVolumes method to apply some per-platform settings on all volumes. 

**The problems** with the current design are that: 

1. It ignores volumes with 0 weight: Usually 0-weight volumes may be animated now or soon, so they are actually "active".
2. It ignores inactive volumes, which might just be the case because objects are being set inactive or spawned inactive and then turned on for performance optimizations/culling. 

**The solution** we have implemented feels a bit hacky, so it might not be ideal, but gives the coder much more flexibility and understanding of what happens under the hood:

```
public void GetActiveVolumes(PostProcessLayer layer, List<PostProcessVolume> results, bool skipInactive = true, bool skipZeroWeight = true)
        {
            // If no trigger is set, only global volumes will have influence
            int mask = layer.volumeLayer.value;
            var volumeTrigger = layer.volumeTrigger;
            bool onlyGlobal = volumeTrigger == null;
            var triggerPos = onlyGlobal ? Vector3.zero : volumeTrigger.position;

            for (int i = 0; i < k_MaxLayerCount; i++)
            {
                // Skip layers not in the mask
                if ((mask & (1 << i)) == 0)
                    continue;

                // Skip empty layers
                var volumes = m_Volumes[i];

                if (volumes == null)
                    continue;

                // Traverse all volumes
                foreach (var volume in volumes)
                {
                    // Skip disabled volumes and volumes without any data or weight
                    if ((skipInactive && !volume.enabled) || volume.profileRef == null || (skipZeroWeight && volume.weight <= 0f))
                        continue;
```
Long story short, we add two parameters with default set to true, so that anyone can override these and have more control over what the method returns.
Cheers!

**Comment by Chman** [comment]
I added this to the mainline, seems like a nice feature to have. By the way, you should update your v2 version as it fixes a bunch of critical issues (one of which is still there in your snippet) and improves CPU performances a lot.

**Comment by AlexandreFiset** [comment]
That was quick :) Thanks!
